app/views/shared/_loss_report_modal.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="loss-report-modal" class="eui-modal-content" >
+  <a href="javascript:void(0);" class="modal-close float-r" id="close-loss-report-modal"><i class="fa fa-times"></i><span class="is-invisible">Close</span></a>
+  <div>
+    <h4>Conversion to UMM Format Required</h4>
+    <p>The native format of this collection is <%= options[:native_format].split('/').last.gsub('+xml','').upcase %>. Editing this record using the MMT will convert it to UMM-JSON, which may result in data loss. Select 'View Loss Report' to see how the conversion will affect this record.</p>
+    <p style="display: inline-flex">
+      <% if current_provider?(options[:provider_id]) %>
```

Comment:
The view loss report and cancel buttons are the same in both of these scenarios and so are repeated.
1. Can the order of the buttons be changed, or is this the order the PO wanted? If it can be changed it would be easier to 'dry' this up.
2. If the order is fixed, I'm debating if it makes more sense to do something like
```
if current_provider || available_provider
  loss report button
  if current_provider
    edit button
  else
    edit button
  end
  cancel button
end
```
That doesn't seem too ideal either. Other opinions would be helpful here.

Reply:
The modal is not visible unless `current_provider || available_provider` is true.  I think it can just be:
```loss report button
  if current_provider
    edit button
  else
    edit button
  end
  cancel button```